packages/quantic/force-app/solutionExamples/main/lwc/exampleSearch/e2e/exampleSearch.e2e.ts

```diff
@@ -0,0 +1,156 @@
+import {testSearch, expect} from './fixture';
+
+let test = testSearch;
+
+test.describe('Example Search Page E2E Tests', () => {
+  test.describe('when loading the search page', () => {
+    test('should render correctly and load results automatically', async ({
+      searchPage,
+      page,
+    }) => {
+      const consoleErrors = [];
+
+      // Listen for console error messages
+      page.on('console', (msg) => {
+        if (msg.type() === 'error') {
+          consoleErrors.push(msg.text());
```

Comment:
You're not doing anything with those, did you forget an `expect` ?

`expect(consoleErrors.length).toBe(0);`

Reply:
🤦 I forgot to add the expect indeed haha I will add it